docs/pages/identity-governance/aws-iam-identity-center/guide.mdx

```diff
@@ -113,6 +115,21 @@ the same [IdP](../../zero-trust-access/sso/sso.mdx) as your current Identity Cen
 external identity source.
 </Admonition>
 
+<Admonition type="info" title="SAML IdP RBAC">
```

Comment:
There are a lot of caveats and colored boxes before we get to the main content of the section, so it's difficult for me to tell what this section is actually trying to get me to do.

<img width="719" height="682" alt="image" src="https://github.com/user-attachments/assets/35192728-f363-422f-97fa-f413ac9f05ba" />

Is there a way to structure the section to begin with an overview of what Step 3 entails, then combine the warnings into a single, concise Admonition?

Reply:
Admonition removed now, see updates  here  https://github.com/gravitational/teleport/pull/58667#discussion_r2325129721